plugin.video.wakanim/addon.xml

```diff
@@ -23,7 +23,7 @@
         <disclaimer lang="de_DE">HINWEIS: Du MUSST PREMIUM Mitglied sein oder Videos KAUFEN um dieses Plugin zu benutzen</disclaimer>
         <disclaimer lang="fr_FR">AVERTISSEMENT: VOUS DEVEZ être un membre PREMIUM ou ACHETER des vidéos pour utiliser ce plugin</disclaimer>
         <disclaimer lang="ru_RU">ВНИМАНИЕ: Для использования этого плагина Вы должны оформить подписку или купить аниме</disclaimer>
-        <news>v0.7.6 (2019.05.06)[CR]- Fix login not working[CR]- Small fixes</news>
+        <news>v0.8.0 (2019.12.01)[CR]- Fix playing with dash proto (thanks ex0hunt)[CR]- Fix search[CR]- Fix second season not opening info-tab[CR]- Fix trailer[CR]- Removed watchlist and collection</news>
         <license>GNU Affero General Public License, v3</license>
```

Comment:
can you please use the SPDX license identifier instead of the full license name. Should be `GPL-3.0` (see: https://spdx.org/licenses/)